lisa/env.py

```diff
@@ -509,6 +509,10 @@ def _init_target(self, target_conf, res_dir):
 
         target.setup()
 
+        # Only available for Android targets
+        with contextlib.suppress(AttributeError):
```

Comment:
Any reason for doing this instead of that:
```python
if target_kind == "android":
    target.adb_root(force=True)
```
?

Reply:
Not especially, the one ignoring the exception does not have to make any assumption on the exact class used